### PR TITLE
Infrastructure updates

### PR DESCRIPTION
**Pull request by sree-cfa**
- Adding "demo" env for auto-deploying with main branch
- tweak to testing for checking for missing message properties

### CODE REVIEW

File: src/main/resources/application-demo.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+spring:
+  thymeleaf:
+    cache: true
+  datasource:
+    url: jdbc:postgresql://host.docker.internal:5432/il-gcc
+form-flow:
+  disabled-flows: ~
```
Comment (coltborg) [inline src/main/resources/application-demo.yaml:7]: What does `~` do? Is it just a placeholder?

File: src/main/resources/application-demo.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  aws:
+    s3_bucket_name: 'get-child-care-illinois-demo'
```
[the remark this answers is not in the document]
Comment (sree-cfa) [reply src/main/resources/application-demo.yaml:9]: Yes - creating it now

File: src/main/resources/application-demo.yaml (hunk 1 above)
Comment (sree-cfa) [reply src/main/resources/application-demo.yaml:7]: Yeah - we probably want to add `ubi` here?

Comment (coltborg) [reply src/main/resources/application-demo.yaml]: I don't think we have to, I'd like to delete `ubi` soonish.

**Review by coltborg — APPROVED** [review]
lgtm;

**Pull request merged**